man/docker-network-connect.1.md

```diff
@@ -0,0 +1,53 @@
+% DOCKER(1) Docker User Manuals
+% Docker Community
+% OCT 2015
+# NAME
+docker-network-connect - connect a container to a network
+
+# SYNOPSIS
+**docker network connect NAME CONTAINER**
+
+[**--help**]
+
+# DESCRIPTION
+
+Connects a running container to a network. You can connect a container by name
+or by ID. Once connected, the container can communicate with other containers in
+the same network.
+
+```bash
+$ docker network connect multi-host-network container1
+```
+
+You can also use the `docker run --net` option to start a container and
+immediately connect it to a network.
+
+```bash
+$ docker run -itd --net=my-bridge-network busybox
```

Comment:
Wonder if this should be the same network name as created above :stuck_out_tongue_closed_eyes: